wrappers/matlab/option.m

```diff
@@ -74,8 +74,8 @@
         sensor_mode                     (70)
         emitter_always_on               (71)
         thermal_compensation            (72)
-        trigger_camera_accuracy_health  (73)
-        reset_camera_accuracy_health    (74)
+        % trigger_camera_accuracy_health  (73)
```

Comment:
I'd rather keep those even if not in use

Reply:
I put them back